src/component/registry/css-registry.ts

```diff
@@ -32,30 +32,64 @@ export class CssRegistry {
   /**
    * Get the CSS class names for a specific HTML element
    *
-   * @param bpmnElementId the BPMN id of the HTML element from the DOM
+   * @param bpmnElementId the BPMN ID of the HTML element from the DOM
    * @return the registered CSS class names
    */
   getClassNames(bpmnElementId: string): string[] {
     return Array.from(this.classNamesByBpmnId.get(bpmnElementId) ?? []);
   }
 
+  /**
+   * Get all BPMN element IDs with at least one CSS class on its HTML element.
+   *
+   * @return an array of string values representing the BPMN element IDs.
```

Comment:
**suggestion:** no need to specify the type of the returned value as it is already documented in the signature